common/src/main/scala/uk/ac/wellcome/dynamo/SourceDataDao.scala

```diff
@@ -0,0 +1,66 @@
+package uk.ac.wellcome.dynamo
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB
+import com.google.inject.Inject
+import com.gu.scanamo.{Scanamo, Table}
+import com.gu.scanamo.syntax._
+import com.gu.scanamo.ops.ScanamoOps
+import com.gu.scanamo.syntax.{attributeExists, not}
+import com.twitter.inject.Logging
+import uk.ac.wellcome.models.aws.DynamoConfig
+
+import uk.ac.wellcome.utils.GlobalExecutionContext.context
+
+import scala.concurrent.Future
+
+case class SourceData(
+  id: String,
+  version: Int,
+  reindexShard: String,
+  reindexVersion: Int,
+  ref: String
```

Comment:
What’s “ref” a reference to? A URL? An S3 object?

Reply:
Will rename in further branch.